packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaGroup.ts

```diff
@@ -97,6 +97,68 @@ describe('RemoteMediaGroup', () => {
     });
   });
 
+  describe('setPreferLiveVideo', () => {
+    it('sets to true initial option was false', () => {
+
+      const group = new RemoteMediaGroup(fakeMediaRequestManager, fakeReceiveSlots, 255, true, {
+        resolution: 'medium',
+        preferLiveVideo: false,
+      });
+
+      group.setPreferLiveVideo(true);
+
+      assert.calledOnce(fakeMediaRequestManager.cancelRequest);
+
+      assert.calledTwice(fakeMediaRequestManager.addRequest);
+
+      assert.calledWith(
+        fakeMediaRequestManager.addRequest,
+        sinon.match({
+          policyInfo: sinon.match({
+            policy: 'active-speaker',
+            priority: 255,
+            preferLiveVideo: true
+          }),
+          receiveSlots: fakeReceiveSlots,
+          codecInfo: sinon.match({
+            codec: 'h264',
+            maxFs: 3600,
+          }),
+        })
+      );
+    });
+
+    it('sets to false initial option was true', () => {
```

Comment:
I don't feel there is much value in having 2 same tests with just the value changed true vs false - for me 1 would be enough, but there is another more interesting case, so maybe we should have these 2 tests: 
1. "setPreferLiveVideo() called with different value",
2. "setPreferLiveVideo() called with same value" - in that case we could detect it and avoid doing anything
Or if you prefer to test all boolean values then we should have tests for all 4 cases (all combinations of initial value and new value) and the tests should be parametrized to avoid duplication